src/views/App.js

```diff
@@ -3,17 +3,49 @@ import './App.css';
 import ServiceContent from './ServiceContent';
 import TextField from '@material-ui/core/TextField';
 import { makeStyles } from '@material-ui/styles';
+import { Button } from '@material-ui/core';
+import Dialog from '@material-ui/core/Dialog';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+
 
 function App() {
+  let storedid = localStorage.getItem('steamId') ? localStorage.getItem('steamId') : '';
   const [values, setValues] = React.useState({
-    steamid: '',
+    steamid: storedid,
     age: '',
+    opened: true,
+    openDialog: false
   });
 
+  const [openDialog, setDialog] = React.useState(false);
+
   const handleChange = name => event => {
     setValues({ ...values, [name]: event.target.value });
   };
 
+  const saveItem = () => {
+    if (!localStorage.getItem('steamId')) {
```

Comment:
Ya, as I said in `line 14`

Reply:
As this returns null, I want to ensure it gets set at this point. I'm going to add a check on the string being saved itself to avoid null being saved as a localstorage value